tools/reviewer/remote_server/README.md

```diff
@@ -0,0 +1,34 @@
+# Remote Server
+
+# Workflow
+
+
+## Initial Auth
```

Comment:
Please add a note that the server has to be HTTPS, so that the code is secure in transit.